blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,225 @@
+contributions to firmware projects. Our primary focus revolved around
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers),
+a firmware solution that captured our attention. We also dedicated efforts to
+the advancement of fwupd and flashrom firmware update systems. But our
+dedication to system security didn't stop there—we actively developed
+[Trenchboot](https://blog.3mdeb.com/tags/trenchboot/), enabling dynamic
+checksum measurement for system components. And let's not forget our deep
+involvement in the intricate world of
+[Yocto](https://3mdeb.com/training/#yocto-project-development), where we left
+our mark on various Yocto layers.
+
+In addition to contributing to these open-source projects, our team is also
+passionate about promoting the benefits of open-source software. We believe
+that open-source software offers numerous advantages over proprietary software,
+such as greater transparency, flexibility, and security. By contributing to
+these projects and advocating for open-source software, we aim to foster
+a culture of collaboration, innovation, and inclusivity in the technology
+industry. We are committed to making a positive impact on the community
+through our work, and we look forward to continuing our open-source
+contributions in the future.
+
+## Our contributions
+
+![coreboot logo](/covers/coreboot-logo.svg)
+
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers)
+is an open-source firmware that provides a lightweight, secure,
+and fast boot experience for PCs, laptops, servers, and embedded devices.
+The changes made by our engineers can be divided into three main groups.
+
+#### Adding support for new platforms and maintaining existing ones
+
+Recently, our engineers have focused on expanding coreboot support for the
+latest platforms, it is worth mentioning that Karol Zmyslowski has added
+[support for Jasper Lake](https://review.coreboot.org/c/coreboot/+/73934).
+
+Michał Kopeć and Michał Żygowski had a major impact on Alder Lake support in
+coreboot. Michał Kopeć has added [support for Alder Lake](https://review.coreboot.org/c/coreboot/+/63374)
+and Michał Żygowski in the [set of patches](https://review.coreboot.org/c/coreboot/+/68449/11)
+has added improvements huge improvements. Also, Michał Żygowski made a significant
+contribution to [Tiger Lake support](https://review.coreboot.org/c/coreboot/+/56171).
+
+In the previous year, our team also added
+[support for MSI](https://review.coreboot.org/c/coreboot/+/68078) PRO Z690-A
+WIFI and MSI PRO Z690-A in DDR4 and DDR5 versions.
+
+Krystian Hebel, Sergii Dmytruk, and Yaroslav Kurlaev have made major contributions
+to developing and maintaining support for Power platforms. Their changes include
+[emulation improvements](https://review.coreboot.org/c/coreboot/+/67061),
+and fixing ongoing issues related to these platforms.
+
+#### Platform security (TPM, TXT)
```

Comment:
```suggestion
#### Platform security (TPM, Intel TXT)
```

Please fix also other instances

Reply:
Deleted this section and now we're linking to the 4.20 release blog. https://github.com/3mdeb/news-and-ideas/commit/68a72cdebcee326365faf61a3b7861f718765122#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR79